src/mapbuffer.h

```diff
@@ -66,6 +67,7 @@ class mapbuffer
         // submap exists or not.
         bool submap_exists( const tripoint_abs_sm &p );
 
+
```

Comment:
You seem to have some not fully cleaned up edits, please make sure your PR doesn't have errant edits like this scattered around.